externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/SparkSQLOperationManager.scala

```diff
@@ -58,7 +61,15 @@ class SparkSQLOperationManager private (name: String) extends OperationManager(n
       runAsync: Boolean,
       queryTimeout: Long): Operation = {
     val spark = getSparkSession(session.handle)
-    val operation = new ExecuteStatement(spark, session, statement, runAsync, queryTimeout)
+
+    val operationMode = {
+      val modeStr = spark.conf.get(OPERATION_PLAN_ONLY.key, NONE.toString).toUpperCase(Locale.ROOT)
```

Comment:
If we set a default value other than `None` in kyuubi server, seems the setting will not work here.

Reply:
updated